packages/kbn-interpreter/src/plugin/types/kibana_datatable.js

```diff
@@ -0,0 +1,44 @@
+/*
```

Comment:
Any objection to switching new files over to typescript?

Reply:
kbn-interpreter doesn't support TypeScript so far. Please interact with https://github.com/elastic/kibana/pull/28954 which also contains setting up TypeScript in the kbn-interpreter package. Also it's the beginning of getting general typings in there. Just having that file here in TS wouldn't bring much benefit, because we have no typings yet for datatable and pointseries formats, which I would like to do as a second step, once that above PR is ready.